File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        countOnes = 0
        maxOnes = 0
        for num in nums:
            if num == 1:
                countOnes += 1
            else:
                countOnes = 0
            
            if countOnes > maxOnes:
                maxOnes = countOnes
        
        return maxOnes

File: Data Structures & Algorithms/remove-element/submission-0.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int start = 0;
        int end = nums.length;
        
        while (start < end) {
            if (nums[start] == val) {
                nums[start] = nums[--end];
            } else {
                start++;
            }

        }

        return end;
    }
}